keps/sig-scheduling/3838-pod-mutable-scheduling-directives/kep.yaml

```diff
@@ -0,0 +1,38 @@
+title: Pod Mutable Scheduling Directives
+kep-number: 3838
+authors:
+  - "@ahg-g"
+owning-sig: sig-scheduling
+status: implementable
+creation-date: 2023-02-03
+reviewers:
+  - "@Huang-Wei"
+  - "@liggit"
+approvers:
+  - "@Huang-Wei"
+
+see-also:
+  - "/keps/sig-scheduling/2926-job-mutable-scheduling-directives"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: beta
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.27"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  beta: "1.27"
+  stable: "v1.28"
+
+feature-gates:
+  - name: PodMutableNodeSchedulingDirectives
```

Comment:
what's the mechanics of this feature gate that depends on another (PodSchedulingReadiness)? Is that under the hood it's initiated as:

```
enabled := <this feature gate enabled> AND <dependent feature gate enabled>
```

Reply:
hmm, I guess we can depend on the same flag, we don't need to introduce a new one.